helm/trivy/values.yaml

```diff
@@ -0,0 +1,100 @@
+nameOverride: ""
+fullnameOverride: ""
+
+image:
+  registry: docker.io
+  repository: aquasec/trivy
+  tag: 0.13.0
+  pullPolicy: IfNotPresent
+
+replicaCount: 1
+
+persistence:
+  enabled: true
+  storageClass: ""
+  accessMode: ReadWriteOnce
+  size: 5Gi
+
+resources:
+  requests:
+    cpu: 200m
+    memory: 512Mi
+  limits:
+    cpu: 1
+    memory: 1Gi
+
+rbac:
+  create: true
+  pspEnabled: true
+
+podSecurityContext:
+  runAsUser: 1000
```

Comment:
For aquasecrity/harbor-scanner-trivy image we created user scanner(id=1000). It's not the case for Trivy image. So maybe you can use nobody(id=65534)